src/iterator/CellListAllTriplesIterator.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+  Copyright (C) 2012,2013
```

Comment:
2019?

Reply:
Not sure, copyright should rather represent full history of the contributors (a.k.a. copyright owners).